CommandBased/src/com/spikes2212/ftc/command/drivetrains/DrivetrainBase.java

```diff
@@ -0,0 +1,58 @@
+package com.spikes2212.ftc.command.drivetrains;
+
+import com.qualcomm.robotcore.hardware.DcMotorSimple;
+import com.spikes2212.ftc.command.Subsystem;
+
+/**
+ * A base class for drivetrain systems.
+ */
+public class DrivetrainBase implements Subsystem {
+
+    /**
+     * The motors on this drivetrain.
+     */
+    private final DcMotorSimple[] motors;
+
+    protected DrivetrainBase(DcMotorSimple... motors) {
+        this.motors = motors;
+    }
+
+    /**
+     * Drives this drivetrain with the given speed on each motor.
+     *
+     * <p>If any of the speeds is greater than 1, the speeds are normalized.</p>
+     *
+     * <p>The order of the speeds is the same as the order of the motors, as given in the constructor.</p>
+     *
```

Comment:
Specify what happens if the length of `speeds` and the length of `motors` differ